s3/replication/common/setup.cfg

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+[metadata]
+name = s3-replication-common
+author = Seagate
+author_email = opensource@seagate.com
+description = Common modules for CORTX cross cluster/region replication for S3 objects.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Seagate/cortx-multisite
+project_urls =
+    Bug Tracker = https://github.com/Seagate/cortx-multisite/issues
```

Comment:
Not sure, why 'Tracker' is showing as red, though not start of the line. May be indentation problem?

Reply:
May be its just recognising it as keyword.